### PR TITLE
Solved merge conflicts: Sidebar navigation

### LINKED ISSUE #491: ✨ Enhancement: Sidebar must be removed automatically when an option is chosen/clicked in mobile view
### Is your feature request related to a problem? Please describe

https://github.com/json-schema-org/website/assets/96608169/248aafe3-ae11-4e18-97de-83b1ee08d8f6

### Describe the solution you'd like

It must remove the sidebar when user clicks on an option. It can be solved in many ways like using state management, etc. 

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Are you working on this?

Yes

### PR DESCRIPTION
**Pull request by VivekJaiswal18**
**What kind of change does this PR introduce?**
Feature

**Issue Number:**
-  Closes #491 

**Screenshots/videos:**
Before:

https://github.com/json-schema-org/website/assets/96608169/a0de038c-af69-40c9-abaa-c9d9528ae0fc

After:

https://github.com/json-schema-org/website/assets/96608169/4251246e-c6bc-463f-ae1f-5992c228c7c8

**If relevant, did you update the documentation?**
No

**Summary**
Now sidebar closes dynamically when user clicks on any option inside the sidenav bar. Which makes it more ergonomic.

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by VivekJaiswal18** [comment]
Hey @benjagm please have a look into this.
Thank you!

**Comment by benjagm** [comment]
This looks great!!! Can you please rebase to make sure there are no conflicts?

**Review by benjagm — CHANGES_REQUESTED** [review]
Looks great. Can you please rebase to make sure we don't have conflicts?

**Comment by VivekJaiswal18** [comment]
Thanks for the feedback @benjagm. Rebasing the commit

**Comment by VivekJaiswal18** [comment]
@benjagm rebased it please have a look.
Thank you

File: yarn.lock [hunk 1]
```diff
@@ -2352,7 +2352,7 @@ next-themes@^0.2.1:
   version "0.2.1"
   resolved "https://registry.yarnpkg.com/next-themes/-/next-themes-0.2.1.tgz#0c9f128e847979daf6c67f70b38e6b6567856e45"
   integrity sha512-B+AKNfYNIzh0vqQQKqQItTS8evEouKD7H5Hj3kmuPERwddR2TxvDSFZuTj6T7Jfn1oyeUyJMydPl1Bkxkh0W7A==
-
+
```
Comment (DhairyaMajmudar) [inline yarn.lock:2355]: From here also : )

**Comment by VivekJaiswal18** [comment]
I think there is some issue with the command `yarn run format:fix` every time I run it, this adds unwanted changes in the code format especially in the global.css file. I think it needs to be addressed is there anyone else facing the same issue.
image
image

**Comment by JeelRajodiya** [comment]
> I think there is some issue with the command `yarn run format:fix` every time I run it, this adds unwanted changes in the code format especially in the global.css file. I think it needs to be addressed is there anyone else facing the same issue. image

I think it is normal, the global.css file was not following the rules defined in the prettier config. When you ran `format:fix` it ran prettier in that file as well 

It changed the tab width to 2 spaces.

**Comment by JeelRajodiya** [comment]
I think you should keep it, so no one else has to tackle it in future.

**Comment by aialok** [comment]
> I think there is some issue with the command `yarn run format:fix` every time I run it, this adds unwanted changes in the code format especially in the global.css file. I think it needs to be addressed is there anyone else facing the same issue.
> image
> 
> 

I think the problem is coming because of this Prettier formatter options.

**Comment by VivekJaiswal18** [comment]
> > I think there is some issue with the command `yarn run format:fix` every time I run it, this adds unwanted changes in the code format especially in the global.css file. I think it needs to be addressed is there anyone else facing the same issue.
> > image
> 
> I think the problem is coming because of this Prettier formatter option.
> 
> 

Yes you are correct beacause everyone is avoiding the these changes it is being carried on

**Review by benjagm — APPROVED** [review]
Great job! Thanks!!

**Pull request merged**